charts/alfresco-adf-app/values.yaml

```diff
@@ -78,6 +78,8 @@ volumes: []
 # -- Additional volumeMounts on the output Deployment definition
 volumeMounts: []
 
+configMap: []
```

Comment:
If configmap is a list then I'm not sure how this works.
Do you expect to render a json list (I think that's not the expected format)? or do you plan to have more than one confimap (in which case we'd need an iterator somewhere)?

Reply:
I was planning just to have one configmap. Whatever you put in yaml format it will be converted to json. I our case it can look like this:
```
configMap:
  $schema: "../../../node_modules/@alfresco/adf-core/app.config.schema.json"
  providers: ALL
  ecmHost: "{protocol}//{hostname}{:port}"
  bpmHost: "{protocol}//{hostname}{:port}"
  authType: OAUTH
  ...
``` 
Let me change that to {}